# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1382779845 -7200
#      Sat Oct 26 11:30:45 2013 +0200
# Node ID 7fc06bb4ab24c65119fd74f731b20aad08d32693
# Parent  26a418799bc367a201dee93c800b352476c52ebf
fr_FR.ts : Update French Translation

diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -239,17 +239,17 @@ est déjà ouvert dans l&apos;éditeur</translation>
     <message>
         <location line="+63"/>
         <source>&amp;Recent Editor Files</source>
         <translation>Fichiers &amp;récents</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Edit Function</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Editer Fonction</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Close</source>
         <translation>&amp;Fermer</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -259,58 +259,64 @@ est déjà ouvert dans l&apos;éditeur</translation>
     <message>
         <location line="+5"/>
         <source>Close Other Files</source>
         <translation>Fermer les autres fichiers</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Run &amp;Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Exécuter la &amp;Selection</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="unfinished">&amp;Aide</translation>
+        <translation>&amp;Aide</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Aide sur le mot de clés</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation type="unfinished"></translation>
+        <translation>Documentation sur le mot de clés</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
         <translation>&amp;Rechercher et remplacer</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossible d&apos;ouvrir le fichier
+%1 
+pour lecture : %2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation type="unfinished"></translation>
+        <translation>Le fichier
+%1
+n&apos;existe pas. Voulez-vous le créer.</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossible d&apos;ouvrir le fichier
+%1 
+pour écrire : %2.</translation>
     </message>
     <message>
         <location line="+549"/>
         <source>Save File And Run</source>
         <translation>Enregistrer et exécuter</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -325,22 +331,22 @@ for write: %2.</source>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
         <translation>&amp;Editer</translation>
     </message>
     <message>
         <location line="+22"/>
         <source>&amp;Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Préférences</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Styles Preferences</source>
-        <translation type="unfinished"></translation>
+        <translation>Préférences de &amp;Style</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Debug</source>
         <translation>&amp;Déboguer</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -1001,17 +1007,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="-1011"/>
         <location line="+931"/>
         <source>Save Workspace As</source>
         <translation>Enregistrer l&apos;espace de travail sous</translation>
     </message>
     <message>
         <location line="-809"/>
         <source>Octave Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Notes de la version d&apos;Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Définir le répertoire de travail</translation>
     </message>
     <message>
         <location line="+736"/>
@@ -1071,27 +1077,27 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+48"/>
         <source>Show File Browser</source>
         <translation>Afficher l&apos;explorateur de fichiers</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Show News Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Afficher Fenêtre des Nouvelles</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>File Browser</source>
         <translation>Explorateur de fichiers</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>News</source>
-        <translation type="unfinished"></translation>
+        <translation>Nouvelles</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Reset Default Window Layout</source>
         <translation>Rétablir la disposition par défaut des fenêtres</translation>
     </message>
     <message>
         <location line="+86"/>
@@ -1121,27 +1127,27 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+3"/>
         <source>Online</source>
         <translation>En ligne</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>&amp;News</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Nouvelles</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Release Notes</source>
-        <translation type="unfinished"></translation>
+        <translation>Notes de la version</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Nouvelles de la Communauté</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
         <translation>Entrez le nom du répertoire</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -1230,17 +1236,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <translation>Signaler un bogue</translation>
     </message>
 </context>
 <context>
     <name>news_dock_widget</name>
     <message>
         <location filename="../src/news-dock-widget.cc" line="+44"/>
         <source>Community News</source>
-        <translation type="unfinished"></translation>
+        <translation>Nouvelles de la Communauté</translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
     <message>
         <location filename="../src/octave-dock-widget.cc" line="+56"/>
         <location line="+159"/>
         <source>Undock widget</source>
@@ -1257,32 +1263,32 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <translation>Attacher le widget</translation>
     </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+127"/>
         <source>Help on</source>
-        <translation type="unfinished"></translation>
+        <translation>Aide sur</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Documentation on</source>
-        <translation type="unfinished"></translation>
+        <translation>Documentation sur</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Edit</source>
-        <translation type="unfinished"></translation>
+        <translation>Editer</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Exécuter la Selection</translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+270"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
         <translation>Le fichier %1 n&apos;existe pas dans les chemins accessibles. Pour déboguer la fonction que vous éditez, vous devez soit modifier le répertoire pour %2 ou ajouter le répertoire aux chemins accessibles.</translation>
@@ -1418,17 +1424,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+61"/>
         <source>Restore editor tabs from previous session on startup</source>
         <translation>Restaurer les onglets de la session précédente</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Create non existing files without prompting</source>
-        <translation type="unfinished"></translation>
+        <translation>Créer des fichiers non existants sans demander</translation>
     </message>
     <message>
         <location line="+47"/>
         <source>Use custom file editor</source>
         <translation>Utiliser un éditeur externe</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1689,38 +1695,51 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
         <translation>Bienvenu dans Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il semble que vous exécutez l&apos;interface graphique d&apos;Octave pour la première fois sur cet ordinateur. Cliquez sur &apos;Fin&apos; pour écrire un fichier de configuration et lancez l&apos;interface graphique d&apos;Octave.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
-        <translation type="unfinished"></translation>
+        <translation>Le fichier de configuration est stockée dans % 1. Si ce fichier existe, vous ne verrez pas cette boîte de dialogue lorsque d&apos;Octave recommence.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;For more information about Octave:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visit &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
 &lt;li&gt;Get the documentation online as &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
 &lt;li&gt;Open the documentation browser of Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head&gt;
+&lt;style&gt;
+a:link { text-decoration: underline; color: #0000ff; }
+&lt;/style&gt;
+&lt;head/&gt;&lt;body&gt;
+&lt;p&gt;Pour plus d'information sur d&apos;Octave :&lt;/p&gt;
+&lt;ul&gt;
+&lt;li&gt;Visiter &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
+&lt;li&gt;Obtenir le documentation en ligne comme &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
+
+;/a&gt;-document&lt;/li&gt;
+&lt;li&gt;Ouvrir le navigateur de documentation de l'interface graphique d'Octave avec le menu d'aide.&lt;/li&gt;
+&lt;/ul&gt;
+&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Finish</source>
         <translation>Fin</translation>
     </message>
 </context>
 <context>
